tools/ci/osquery-ubuntu18.04-toolchain.dockerfile

```diff
@@ -55,4 +67,5 @@ RUN rm -rf /var/lib/apt/lists/*
 # image.
 FROM scratch AS builder
 COPY --from=base3 / /
+COPY --from=cppcheck /root/cppcheck/install/usr/local/ /usr/local/
```

Comment:
This adds a second `COPY`, which I think counts as a second layer. Somewhat mixed on that.

I'd probably:
```
FROM base3 as base4
COPY --from=cppcheck /root/cppcheck/install/usr/local/ /usr/local/

FROM scratch as build
COPY --from=base4 / /
```

Reply:
This is me not knowing enough Docker and having the wrong expectations; I'll change it.

Reply:
I don't love this architecturally, but AFAIK it's what we get